Mathlib/Data/Set/NAry.lean

```diff
@@ -297,6 +297,10 @@ theorem image2_right (h : s.Nonempty) : image2 (fun _ y => y) s t = t := by
   simp [nonempty_def.mp h, ext_iff]
 #align set.image2_right Set.image2_right
 
+lemma image2_range (f : α' → β' → γ) (g : α → α') (h : β → β') :
```

Comment:
Maybe call it a theorem to match the other names in this file? (although I can't remember what we ended up deciding with theorem vs lemma)

Reply:
While we don't have a formal coding style here, I follow an informal "important theorems are theorems, trivial are lemmas" style. Other statements in this file are theorems because of mathport.